config/config.go

```diff
@@ -153,6 +153,10 @@ type Local struct {
 	// Valid units are 's' seconds, 'm' minutes, 'h' hours
 	LogArchiveMaxAge string `version[4]:""`
 
+	// Log to console out only.
+	// Used when running node within docker container
+	LogToConsole bool `version[13]:"false"`
```

Comment:
Could you implement this feature in a slightly different way ?
Instead of having the `LogToConsole`, add `LogFileName`. If the LogFileName is "-", then set the output to StdOut. Otherwise, write it to file ( i.e. as is ).

This would address the issue of having a hard-coded named log file, allowing redirection to stdout, just like the goal commands. When doing so, you'll also need to take care of the carpenter so that it would pick the correct file. ( i.e. use the config package to load the config.json file, etc. )

Reply:
Thank you for your feedback Great idea, will do so tomorrow evening CEST.